### PR TITLE
move all uploads to upload_files script

### PR DESCRIPTION
**Pull request by IsaackMwenda**

### CODE REVIEW

File: run_scripts/run_pipeline.sh [hunk 1]
```diff
@@ -24,21 +24,22 @@ PERFORMANCE_LOGS_DIR=${10}
 
 RUN_ID=$(date -u +"%Y-%m-%dT%H:%M:%SZ")
 
-./1_coda_get.sh "$CODA_PULL_CREDENTIALS_PATH" "$CODA_TOOLS_ROOT" "$DATA_ROOT"
+#./1_coda_get.sh "$CODA_PULL_CREDENTIALS_PATH" "$CODA_TOOLS_ROOT" "$DATA_ROOT"
```
Comment (as2388) [inline run_scripts/run_pipeline.sh]: I don't think you meant to commit this

Comment (as2388) [inline run_scripts/run_pipeline.sh]: I don't think you meant to commit this

File: upload_files.py [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+    parser.add_argument("run_id", metavar="run-id",
+                        help="Identifier of this pipeline run")
+    parser.add_argument("memory_profile_file_path", metavar="memory-profile-file-path",
+                        help="Path to the memory profile log file to upload")
+    parser.add_argument("data_archive_file_path", metavar="data-archive-file-path",
+                        help="Path to the data archive file to upload"),
+    parser.add_argument("production_csv_input_path", metavar="production-csv-input-path",
+                        help="Path to a CSV file with raw message and demographic response, for use in "
+                             "radio show production"),
+    parser.add_argument("csv_by_message_input_path", metavar="csv-by-message-input-path",
```
Comment (as2388) [inline upload_files.py]: While we're here, we could rename this to mesage_csv_input_path. I think that reads better, and is consistent with the phrasing for the production CSV

File: upload_files.py [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+
+    if pipeline_configuration.drive_upload is not None:
+        log.info(f"Downloading Google Drive service account credentials...")
+        credentials_info = json.loads(google_cloud_utils.download_blob_to_string(
+            google_cloud_credentials_file_path, pipeline_configuration.drive_upload.drive_credentials_file_url))
+        drive_client_wrapper.init_client_from_info(credentials_info)
+
+    if pipeline_run_mode == "all-stages":
+        # Upload to Google Drive, if requested.
+        # Note: This should happen as late as possible in order to reduce the risk of the remainder of the pipeline failing
```
Comment (as2388) [inline upload_files.py]: This note is irrelevant now

File: upload_files.py [hunk 4]
```diff
@@ -0,0 +1,123 @@
...
+        # after a Drive upload has occurred. Failures could result in inconsistent outputs or outputs with no
+        # traced data log.
+        if pipeline_configuration.drive_upload is not None:
```
Comment (as2388) [inline upload_files.py]: I don't think there's a need to separate these if statements anymore.

File: docker-run-upload-logs.sh [hunk 5]
```diff
@@ -2,12 +2,14 @@
```
Comment (as2388) [inline docker-run-upload-logs.sh]: Rename to docker-run-upload-files.sh

File: upload_files.py [hunk 6]
```diff
@@ -0,0 +1,123 @@
...
+    parser.add_argument("csv_by_message_input_path", metavar="csv-by-message-input-path",
+                        help="Path to analysis dataset CSV where messages are the unit for analysis (i.e. one message per row)"),
+    parser.add_argument("csv_by_individual_input_path", metavar="csv-by-individual-input-path",
+                        help="Path to analysis dataset CSV where respondents are the unit for analysis (i.e. one respondent "
+                             "per row, with all their messages joined into a single cell)"),
+    parser.add_argument("pipeline_run_mode", help="whether to generate analysis files or not", choices=["all-stages", "auto-code-only"])
+
+    args = parser.parse_args()
+
+    csv_by_message_drive_path = None
```
Comment (as2388) [inline upload_files.py]: Where does this get used?

File: upload_files.py [hunk 7]
```diff
@@ -0,0 +1,123 @@
...
+                             "credentials bucket")
+    parser.add_argument("pipeline_configuration_file_path", metavar="pipeline-configuration-file-path",
+                        help="Path to the pipeline configuration json file")
+    parser.add_argument("run_id", metavar="run-id",
+                        help="Identifier of this pipeline run")
+    parser.add_argument("memory_profile_file_path", metavar="memory-profile-file-path",
+                        help="Path to the memory profile log file to upload")
+    parser.add_argument("data_archive_file_path", metavar="data-archive-file-path",
+                        help="Path to the data archive file to upload"),
+    parser.add_argument("production_csv_input_path", metavar="production-csv-input-path",
```
Comment (as2388) [inline upload_files.py]: Maybe reorder these to match upload order

File: run_scripts/run_pipeline.sh (hunk 1 above)
Comment (IsaackMwenda) [reply run_scripts/run_pipeline.sh]: this was accidentally committed. Fixed

**Pull request merged**